eitprocessing/binreader/sequence.py

```diff
@@ -183,7 +183,7 @@ def merge_attribute(attr: str) -> list:
     @classmethod
     def from_path(  #pylint: disable=too-many-arguments, unused-argument
         cls,
-        path: Path | str | List[Path | str],
+        path: Path | str | List[Path] | List[str],
```

Comment:
I will drop this change, because there is no inherent problem with a mixed list. Users can give some path in `Path` format and others in `str` format and it will parse correctly.